### ISSUE TITLE
[Accessibility][Sourcing]: fdp tree table accessibility issues

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9132]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
1) Not able to navigate into child row(s) after expanding parent row using down arrow key in tree table
2) Tree level is not output by screenreader

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 13.x.x
   "@fundamental-ngx/core": "0.35.4",
    "@fundamental-ngx/platform": "0.35.4",

#### If this is a bug, please provide steps for reproducing it.
1) Not able to navigate into child row(s) after expanding parent row using down arrow key in tree table
2) Tree level is not output by screenreader
3) Position of the cell should be read out by screenreader eg : column 3 row 4 both in voiceover and jaws (forms mode and linear reading mode)

#### Please provide relevant source code if applicable.
Reproducible on fdp tree table on https://sap.github.io/fundamental-ngx/#/platform/table 

#### Is there anything else we should know?
This is a blocker for Deutsche Telecom customer for guided sourcing

**Comment by I543348** [comment]
This is fixed
